proto/decentraland/social/friendships/friendships.proto

```diff
@@ -85,30 +78,74 @@ message UpdateFriendshipPayload {
   optional Payload auth_token = 2;
 }
 
-message UpdateFriendshipResponse { FriendshipEventResponse event = 1; }
-
-message SubscribeFriendshipEventsUpdatesResponse {
-  repeated FriendshipEventResponse events = 1;
-}
-
 message Payload {
   // For internal use only, subject to change.
   optional string synapse_token = 1;
 }
 
+message BadRequestError {
+  string message = 1;
+}
+message UnauthorizedError {
+  string message = 1;
+}
+message ForbiddenError {
+  string message = 1;
+}
+message TooManyRequestsError {
+  string message = 1;
+}
+message InternalServerError {
+  string message = 1;
+}
+
+message UsersResponse {
+  oneof response {
+    Users users = 1;
+    InternalServerError internal_server_error = 2;
+    UnauthorizedError unauthorized_error = 3;
+  }
+}
+
+message RequestEventsResponse {
+  oneof response {
+    RequestEvents events = 1;
+    InternalServerError internal_server_error = 2;
+    UnauthorizedError unauthorized_error = 3;
+  }
+}
+
+message UpdateFriendshipResponse {
+  oneof response {
+    FriendshipEventResponse event = 1; // TODO: why do we need this?
```

Comment:
In the current flow I think it's because Unity needs the info of the event to update its UI and states. Not sure if Joni's new client could help us improve things and get rid of it 🤞🏼